index.html

```diff
@@ -640,15 +572,8 @@ <h2>
         the angle, so the fold angle value is changing in bursts: the events may be dispatched at a very low
         frequency most of the time and fire at a high frequency when the device is being opened or closed.
         In order for the events to be dispatched, the content must be on the foreground and visible to the
-        user.
-      </p>
-      <p>
-        Minimizing the accuracy of the angle readout generally decreases the risk of fingerprinting.
-        User agents should not provide unnecessarily verbose readouts of the hinge angle data.
-        Posture values are not very useful values to fingerprint a user. They’re similar in concept
-        with the orientation values. The posture change is only triggered in certain fold angle values
-        providing a very low resolution reading. Because the changes are very sparse changes with very
-        few predefined values, it makes it hard to precisely identify users across sites.
+        user. Because we do not dispatch the raw angle values and only the postures, fingerprinting possibilities
```

Comment:
```suggestion
        user. Because the API does not directly expose the raw angle values, but a more abstract posture with fewer possible states, fingerprinting possibilities
```